modules/dop/component-protocol/components/project-pipeline-exec-list/myPage/render.go

```diff
@@ -0,0 +1,23 @@
+// Copyright (c) 2021 Terminus, Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package myPage
+
+import (
+	"github.com/erda-project/erda-infra/providers/component-protocol/cpregister/base"
+)
+
+func init() {
+	base.InitProvider("project-pipeline-exec-list", "myPage")
```

Comment:
no need this file.
please reference to https://yuque.antfin-inc.com/dice/zs3zid/uqgglr#qRTJf

Reply:
done